libs/api/domains/cms/src/lib/search/contentful.service.ts

```diff
@@ -109,25 +108,26 @@ export class ContentfulService {
 
     // write sync token to elastic here as it's own type
     logger.info('Writing next sync token to elasticsearch index')
-    return this.elasticService.index(
-      SearchIndexes[locale],
-      nextSyncTokenDocument,
-    )
+    return this.elasticService.index(elasticIndex, nextSyncTokenDocument)
   }
 
+  interface
```

Comment:
interface alone?

Reply:
Hmm intresting that this ran without troubles, removed